src/main/java/men/groupiron/DataManager.java

```diff
@@ -89,14 +89,16 @@ public void submitToApi() {
 
             Map<String, Object> updates = new HashMap<>();
             updates.put("name", playerName);
+            if (locationOption()) {
```

Comment:
I am looking to accept this, but I want to also send the option selection to the api so it can disable sending the data from there. I would also like to indicate on the website that a player has disabled this. I will try to add this in a few days.

Reply:
@christoabrown Have you been able to add this to the API? <3